git_machete/git_operations.py

```diff
@@ -937,3 +937,10 @@ def display_diff(self, fork_point: AnyRevision, format_with_stat: bool, branch:
     def update_head_ref_to_new_hash_with_reflog_subject(self, hash: FullCommitHash, reflog_subject: str) -> int:
         self.flush_caches()
         return self._run_git("update-ref", "HEAD", hash, "-m", reflog_subject)
+
+    def get_indent_len_before_branch_name(self) -> int:
+        indent_len_before_branch_name = self.get_config_attr_or_none('machete.status.branch.indent.len')
+        if indent_len_before_branch_name:
+            return int(indent_len_before_branch_name)
```

Comment:
Hmm this crashes git-machete in case the length is NOT a valid int... pls check how git behaves in case of such malformated configs, but I'd say a warning + proceeding with `0` would be the best option

Reply:
@PawelLipski Hmm, actually this is already a pretty special case so after some thought i think its unnecessary for it to be int and a boolean flag should be enough, wdyt?

Reply:
Yes, actually even better! let's make it a boolean: whether to add a single space or not